File: Örnek-6.15/C#/Devre.cs
using System;
using System.Runtime.Serialization;


double gerilimK,R1,R2,R3,R4,akim,vr1,pr2;

    Console.WriteLine("gerilim değerini giriniz (volt): ");
gerilimK=Convert.ToDouble(Console.ReadLine());
Console.WriteLine("R1 değerini giriniz (ohm): ");
R1 = Convert.ToDouble(Console.ReadLine());
Console.WriteLine("R2 değerini giriniz (ohm): ");
R2 = Convert.ToDouble(Console.ReadLine());
Console.WriteLine("R3 değerini giriniz (ohm): ");
R3 = Convert.ToDouble(Console.ReadLine());
Console.WriteLine("R4 değerini giriniz (ohm): ");
R4 = Convert.ToDouble(Console.ReadLine());

akim=gerilimK/(R1+R2+R3+R4);
vr1 = R1 * akim;
pr2 = akim * akim * R2;

Console.WriteLine("devreden akan akım: " + akim + "amper");
Console.WriteLine("R1 direnci üzerinde düşen gerilim: " + vr1 + " volt");
Console.WriteLine("R2 direncinin harcadığı güç "+pr2+"watt");

File: Örnek-6.15/python/devre.py


gerilimK=float(input("gerilim değerini giriniz:"))
R1=float(input("R1 direncinin değerini giriniz:"))
R2=float(input("R2 direncinin değerini giriniz:"))
R3=float(input("R3 direncinin değerini giriniz:"))
R4=float(input("R4 direncinin değerini giriniz:"))

akim=gerilimK/(R1+R2+R3+R4)
vr1=akim*R1
pr2=akim*akim*R2

print("devreden akan akım: %0.2f\n"%(akim))
print("R1 direncine düşen gerilim: %0.2f\n"%(vr1))
print("R2 direncinin harcadığı güç: %0.2f\n"%(pr2))